src/darwin/Framework/CHIPTests/MTRDeviceTests.m

```diff
@@ -1564,6 +1574,12 @@ - (void)test017_TestMTRDeviceBasics
         [resubscriptionGotReportsExpectation fulfill];
     };
 
+    // Set the onDeviceConfigurationChanged callback but do nothing. We shouldn't be called.
```

Comment:
```suggestion
    // Make sure there are no device configuration changed notifications during this test.
    // There is nothing changing about the server that would lead to those.
```